packages/stylelint-config-copilot-base/index.js

```diff
@@ -75,9 +75,6 @@ module.exports = {
     // Disallow invalid position @import rules within a stylesheet.
     'no-invalid-position-at-import-rule': null,
 
-    // Disallow units for zero lengths.
-    'length-zero-no-unit': true,
```

Comment:
Instead of removing the rule, please set to `null`.

The rulesets include all rules but are either enabled or disabled.